### PR TITLE
Person API Building

### LINKED ISSUE #1: Create User and Crud Operation
**### **CRUD Operation for User Entity****
```
get : /users ---> Lists of Users
get : /users/{uid}       ----> particular user
post: /users               ---->save user
put : /users/{uid}       ----> update existing user
delete: /users/{uid}    ----> delete particular user
```

### PR DESCRIPTION
**Pull request by Pratikburnwal20**
@anish8129  Please review and merge

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,6 +28,14 @@
...
 			<optional>true</optional>
 		</dependency>
+		
+		<!-- https://mvnrepository.com/artifact/org.springframework.data/spring-data-jpa -->
+    <dependency>
+       <groupId>org.springframework.data</groupId>
+       <artifactId>spring-data-jpa</artifactId>   
+    </dependency>
+		
+
```
Comment (anish8129) [inline pom.xml]: Remove Empty Spaces
Format it

File: src/main/java/io/org/addtocart/controller/PersonController.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+public class PersonController implements PersonRestController{
+
+	@Autowired
+	private PersonService  personservice;
+	@Override
+	public List<Person> findAll() {
+		
+		return personservice.findAll();
+	}
+
```
Comment (anish8129) [inline src/main/java/io/org/addtocart/controller/PersonController.java]: Extra space

File: src/main/java/io/org/addtocart/controller/PersonController.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import io.org.addtocart.service.PersonService;
+
+@Controller
+public class PersonController implements PersonRestController{
+
+	@Autowired
+	private PersonService  personservice;
+	@Override
+	public List<Person> findAll() {
+
```
Comment (anish8129) [inline src/main/java/io/org/addtocart/controller/PersonController.java]: Extra space

**Comment by anish8129** [comment]
Code looks 👍 
But needs formatting.

**Comment by Pratikburnwal20** [comment]
looks good

**Pull request merged**